google-cloud-spanner-hibernate-performance-testing/src/main/java/com/google/cloud/spanner/hibernate/ClientLibraryDriver.java

```diff
@@ -0,0 +1,27 @@
+package com.google.cloud.spanner.hibernate;
+
+import static com.google.cloud.spanner.hibernate.BenchmarkUtil.benchmark;
+
+public class ClientLibraryDriver {
+
+  public static void main(String[] args) {
+    ClientLibraryOperations clientLibraryOperations = new ClientLibraryOperations();
+
+    benchmark(clientLibraryOperations::deleteTestDatabase, "Delete the test DB.");
```

Comment:
Before start of test, database may or may not exist, so benchmark data is not super meaningful. Maybe just delete the database here (to guarantee clean slate) without benchmarking, and then delete again with benchmarking as the last step of this method?

Reply:
Done. Ultimately I decided to remove create/delete database from the benchmarking and just put it in a setup block because hibernate will not be used for creating/deleting the database, so there would be nothing to compare to.